server/constants.py

```diff
@@ -28,3 +28,22 @@ class PluginSettings:
     VIDEO_AUTO_SET = 'video.auto_set'
     VIDEO_MAX_THUMBNAIL_FILES = 'video.max_thumbnail_files'
     VIDEO_MAX_SMALL_IMAGE_SIZE = 'video.max_small_image_size'
+
+
+class JobStatus:
+    """Deferred loading of Girder's JobStatus constants"""
```

Comment:
The objects in this scope are evaluated/initialized on plugin load, where I can't yet take for granted that the code and objects for dependent plugins are initialized and ready for use.

Indeed, I actually tested this and found that my plugin fails to load because
```Python
girder.plugins.jobs.constants
```
fails to import.

So, I have to defer the import until after the plugin is loaded, which other plugins, like large_image, have done.   I just tried to do it in a way I thought was more pythonic; wrapping the details in a proxy object that lazy-imported only when necessary, and well after plugin load.